neon_api_proxy/services/alpha_vantage_api.py

```diff
@@ -48,10 +48,10 @@ class AlphaVantageAPI(CachedAPI):
     API for querying Alpha Vantage.
     """
 
-    def __init__(self, api_key: str = None):
+    def __init__(self, api_key: str = None, cache_seconds: int = 300, **_):
```

Comment:
I don't think I've ever seen `**_` before, but I'm not mad about it

Reply:
It suppresses the unused variable warnings about `**kwargs` when `kwargs` is never referenced. I've used `def fn(*_,** __)` in some websocket callback methods before where I want to catch events like connections but don't really care about any of the call args

Reply:
I don't usually get warnings for *args and **kwargs, or maybe I've just trained myself to ignore them. Good solution though